subsys/mgmt/mcumgr/transport/Kconfig.lorawan

```
@@ -0,0 +1,93 @@
+#
+# Copyright (c) 2024, Jamie McCrae
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# The Kconfig file is dedicated to the LoRaWAN transport of MCUmgr
+# subsystem and provides Kconfig options to control aspects of
+# the transport.
+#
+# Options defined in this file should be prefixed:
+#  MCUMGR_TRANSPORT_LORAWAN_
+
+menuconfig MCUMGR_TRANSPORT_LORAWAN
+	bool "LoRaWAN MCUmgr SMP transport"
+	depends on LORAWAN
+	help
+	  Enables handling of SMP commands received over LoRa.
+
+if MCUMGR_TRANSPORT_LORAWAN
+
+config MCUMGR_TRANSPORT_LORAWAN_FRAME_PORT
+	int "LoRaWAN SMP frame port"
+	range 0 120
```

Port 0 is not allowed for applications, it's reserved for MAC commands. The allowed range for applications is 1-223, see [here](https://resources.lora-alliance.org/technical-specifications/lorawan-specification-v1-1).
```suggestion
	range 1 223
```

---

Fixed